test/main.fmf

```diff
@@ -0,0 +1,30 @@
+# Only common dependencies that are NOT required to run podman-tests.sh are
+# specified here. Everything else is in podman-tests.sh.
+require:
+    - make
+    - policycoreutils
+
+/basic_check:
+    summary: Run basic checks
+    test: make basic_check
+    tag: always
```

Comment:
```suggestion
    tag: [ downstream, upstream ]
```

This also works ;)

Reply:
lol yes, when I tried that none of the tests showed up downstream, hence this experiment :D